src/main/resources/templates/fragments/_bookBrowser.html

```diff
@@ -1,4 +1,4 @@
-<div th:fragment="dataTable" th:switch="${books}" xmlns:th="http://www.w3.org/1999/xhtml">
+<div th:fragment="dataTable" th:switch="${book}" xmlns:th="http://www.w3.org/1999/xhtml">
```

Comment:
Why did you rename it?

Reply:
because in the book controller, we are adding the books to "book" key in the model in books

Reply:
I think this should be changed to books.

Reply:
It should stay books
it was named books since it returns a list of books not a single book

Reply:
unrelated to this pr anyways, if we want to do renaming like this it should be done in a separate pr

Reply:
this was changed because the books are added to the model under the "book" key while we are refeering to the "books" key. In that case, I will change the key in the model and leave this unchanged.

Reply:
The change was moved to the book controller to minimize the number of changes